Working with data: Add 6 month warning (#206)
AlexsLemonade/training-modules: virtual-setup/working-with-your-own-data.md

> - As always, please Slack one of the CCDL team members if you need help with anything (that is what we are here for!).
> 
> - You will have access to our RStudio Server for 6 months.
> We will email you with a warning 6 months from now so you make sure to remove any files from our RStudio Server before we revoke your access and delete them. 

Reviewer: ```suggestion
- You will have access to our RStudio Server for 6 months.
We will email you with a reminder 6 months from now so you can make sure to remove any files from our RStudio Server that you may find useful before your access is revoked and the files are deleted. 

- As always, please Slack one of the CCDL team members if you need help with anything (that is what we are here for!).
```

That is my attempt at softening the language here a bit.

I switched the order of the last 2 bullet points because the bullet point that begins with `As always..` seems like it should be the closing point here.

What do you think?
